### PR TITLE
Add more comments about formatting.

### PR DESCRIPTION
**Pull request by He-Pin**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -167,6 +167,20 @@ sbt
...
 ```
+And you can format the Scala source code with keeping the *import*s sorted with:
+
+```
+sbt
+project akka-actor
+sortImports
+```
+
+To format the Java source code:
```
Comment (patriknw) [inline CONTRIBUTING.md:178]: shouldn't this example be moved up, after the "To format the Scala source code"?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -167,6 +167,20 @@ sbt
 akka-cluster/scalafmtAll
 akka-persistence/scalafmtAll
 ```
+And you can format the Scala source code with keeping the *import*s sorted with:
```
Comment (patriknw) [inline CONTRIBUTING.md:170]: is this comment correct? Looks like it sorts the imports without formatting the Scala source?

Comment (patriknw) [reply CONTRIBUTING.md:170]: @hepin1989 ping?

**Review by octonato — APPROVED** [review]
LGTM

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -169,6 +169,22 @@ akka-cluster/scalafmtAll
 akka-persistence/scalafmtAll
 ```
 
+To format the Java source code:
+```
+sbt
+project akka-actor
+javafmtAll
+```
```
Comment (ignasi35) [inline CONTRIBUTING.md:177]: Actually, [`sbt applyCodeStyle`](https://github.com/akka/akka/blob/master/build.sbt#L27) is a task that will handle headers, scalafmt and javafmt at once. Maybe we should promote that.

**Review by ignasi35 — APPROVED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (hepin1989) [reply CONTRIBUTING.md:177]: I will update with that.

Comment (hepin1989) [reply CONTRIBUTING.md:177]: @ignasi35 I have updated it.

Comment (ignasi35) [reply CONTRIBUTING.md:177]: Brilliant, thanks!

**Pull request merged**